server/cli/cli.py

```diff
@@ -0,0 +1,12 @@
+import click
+
+from .launch import launch
+
+
+@click.group(name='cellxgene', context_settings=dict(max_content_width=85))
+@click.version_option(version='0.0.1', prog_name='cellxgene', message='[%(prog)s] Version %(version)s')
```

Comment:
`setup.py` also has a version. I'd like to use one of these solutions so our versions don't get out of sync: https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
since we are mandating that the package be installed, I suspect `pkg_resources` is a good way to manage this. (see https://github.com/freeman-lab/showit/pull/5 for an example).

Reply:
sounds right to me, will call that out when i open the issue